charts/argocd-image-updater/values.yaml

```diff
@@ -116,7 +116,10 @@ authScripts:
   scripts: {}
     # auth1.sh: |
     #   #!/bin/sh
-    #   echo "auth script 1 here"
+    #  Use when serviceaccount is not working
+    #  aws configure set aws_access_key_id <key>
+    #  aws configure set aws_secret_access_key <secret-key> 
```

Comment:
We should not encourage our users to store secret data inside ConfigMaps...

Are there other options?